portal/messages/es.json

```
@@ -177,6 +177,14 @@
     "demos": "Demos",
     "explorer": "Explorador",
     "get-started": "Comenzar",
+    "help": {
+      "language": "Idioma",
+      "legal-and-privacy": "Legal y Privacidad",
```

I think this sounds better in spanish

```suggestion
      "legal-and-privacy": "Privacidad y Legales",
```

- Plural form
- Reversing the order (it sounds unusual to use Legales first in Spanish). However, in English the current order works better!

---

fixed